Change maximum wifi listed and add an option for trunk(4)

bigio opened the pull request:
Change maximum to 40, useful in some hotels
Add a [-n] optional parameter, dhcp will not be invoked, to be used with trunk(4)

bigio committed:
Change maximum to 40, useful in some hotels

bigio committed:
Change maximum to 40, useful in some hotels

bigio committed:
Change maximum to 40, useful in some hotels

bigio committed:
Change maximum to 40, useful in some hotels

bigio committed:
Add a [-n] optional parameter, dhcp will not be invoked,
to be used with trunk(4)

brycied00d commented:
I like 5c91b0ebf7b7c1169ce507a9eb0097ebf3acc1b6 and, at least until paging functionality is added, believe it should be merged.

The addition of the `-n` parameter should be split into its own feature pullreq.

bigio committed:
Add a [-n] optional parameter, dhcp will not be invoked,  to be used with trunk(4) (#9)

Add a [-n] optional parameter, dhcp will not be invoked,  to be used with trunk(4)

bigio committed:
Merge branch 'master' of https://github.com/bigio/wiconfig

bigio committed:
Merge branch 'master' of https://github.com/bigio/wiconfig

bigio committed:
Merge branch 'master' of https://github.com/bigio/wiconfig

qbit commented:
Not sure if you still have this repo, but could you squash some of the commits? :D

bigio commented:
I do not have this repo anymore.

brycied00d commented:
At least Github can do one big squash.
